Formula/tfsec.rb

```diff
@@ -0,0 +1,17 @@
+class Tfsec < Formula
+  desc "Static analysis powered security scanner for your terraform code"
+  homepage "https://github.com/tfsec/tfsec"
+  url "https://github.com/tfsec/tfsec/archive/v0.30.1.tar.gz"
+  sha256 "95c0b856c95185315cdad4b8442b65cd8178664ed36d776813230d2fb15c43ac"
+  license "MIT"
+  depends_on "go" => :build
+
+  def install
+    system "scripts/install.sh", "v0.30.1"
+    bin.install "tfsec" => "tfsec"
+  end
+
+  test do
+    system "#{bin}/tfsec", "--help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I've updated the test to pull down a sample terraform file and run in the test temp dir. `tfsec` runs against directories so I've run it against `.` - asserting a match on `No problems detected!` which is the success output for `tfsec`